### PR TITLE
Stops the hivemind from getting caugh ton razorwires

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
Fixes good
## Why It's Good For The Game
Bugs bad

## Changelog
:cl:
fix: fixed hivemind eyes getting caught onto razorwires by adding a check
/:cl:

### CODE REVIEW

File: code/game/objects/structures/razorwire.dm [hunk 1]
```diff
@@ -45,6 +45,8 @@
 	if(CHECK_BITFIELD(O.flags_pass, PASSSMALLSTRUCT))
 		return
 	var/mob/living/M = O
+	if(isxenohivemind(M))
+		return
```
Comment (BraveMole) [inline code/game/objects/structures/razorwire.dm:49]: bad, there are status flag like incorporal

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/razorwire.dm (hunk 1 above)
Comment (TiviPlus) [reply code/game/objects/structures/razorwire.dm:49]: Check against flags

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/razorwire.dm (hunk 1 above)
Comment (lbnesquik) [reply code/game/objects/structures/razorwire.dm:49]: Done.

**Comment by Ieteihowmee** [comment]
Wait they get stuck on razor wire??

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**